sb_vision/tokens.py

```diff
@@ -183,10 +200,12 @@ def from_apriltag_detection(
         arr = [apriltag_detection.H.data[x] for x in range(9)]
         homography = np.reshape(arr, (3, 3))
 
+        marker_id = apriltag_detection.id
         instance.infer_location_from_homography_matrix(
             homography_matrix=homography,
             distance_model=distance_model,
             image_size=image_size,
+            marker_size=MARKER_SIZES[marker_id] if marker_id in MARKER_SIZES else None,
```

Comment:
This could be specified much more simply as `MARKER_SIZES.get(marker_id)` (fixed in 4a36718fc638aa46873ef2592a0caaffb30fe1cd)